OsmAnd-java/src/main/java/net/osmand/router/ExitInfo.java

```diff
@@ -15,11 +16,19 @@ public void setRef(String ref) {
         this.ref = ref;
     }
 
-    public String getExitStreetName() {
-        return exitStreetName;
+    public String getDestinationRef() {
+        return destinationRef;
     }
 
-    public void setExitStreetName(String exitStreetName) {
-        this.exitStreetName = exitStreetName;
+    public void setDestinationRef(String destinationRef) {
+        this.destinationRef = destinationRef;
+    }
+
+    public String getDestinationName() {
```

Comment:
destination name has nothing to do with exit info (which is related only to motorway_exit), so it shouldn't be stored here.

Note: destination + destination ref should work on all points even not on motorway junction and it used to work

Reply:
Ok